gui/pages/hostname.go

```diff
@@ -47,7 +47,7 @@ func NewHostnamePage(controller Controller, model *model.SystemInstall) (Page, e
 	page.box.PackStart(page.entry, false, false, 0)
 
 	// Rules label
-	rulesText := utils.Locale.Get("You can use alpha-numeric characters and hyphens with a maximum of 63 characters")
+	rulesText := utils.Locale.Get("Can use alpha-numeric characters and - with a maximum of %d characters.", hostname.MaxHostnameLength)
```

Comment:
Without the "You" this seems strange.
What about:
"Valid characters are alpha-numerics and hyphens '-' with a maximum of %d characters."
or even:
"Valid characters are alpha-numerics and '-' with a maximum of %d characters."

Reply:
Or if for consistency we leave the "Can use", we should at least single quote any characters we reference like:
 "Can use alpha-numeric characters and '-' with a maximum of %d characters."

Reply:
@mhorn Or maybe like the warning msg I can explicitly mention the field name.. E.g "Hostname can contain..."